server_tests/test/clientTests.js

```diff
@@ -1024,27 +1028,25 @@ describe('RUST API TEST SUITE', function () {
     });
 
     it('Expect success. camera.setOptions successfully sets options when fileFormat option is set to supported value raw for image', function () {
-      return testClient.setOptions(sessionId, {
-        fileFormat: {
-          type: 'raw',
-          width: 3840,
-          height: 3840
-        }
-      })
-        .then((res) => validate.done(res.body, schema.names.commandSetOptions),
-          wrapError);
+      if (isBubl1) {
+        return testClient.setOptions(sessionId, {fileFormat: {type: 'raw', width: 3840, height: 3840}})
+          .then((res) => validate.done(res.body, schema.names.commandSetOptions),
+            wrapError);
+      } else if (isBubl2) {
+        return this.skip();
+      }
     });
 
     it('Expect success. camera.setOptions successfully sets options when fileFormat option is set to supported value jpeg for image', function () {
-      return testClient.setOptions(sessionId, {
-        fileFormat: {
-          type: 'jpeg',
-          width: 3840,
-          height: 3840
-        }
-      })
-        .then((res) => validate.done(res.body, schema.names.commandSetOptions),
-          wrapError);
+      if (isBubl1) {
+        return testClient.setOptions(sessionId, {fileFormat: {type: 'jpeg', width: 3840, height: 3840}})
```

Comment:
It would probably make sense to pull out the `fileFormat` into a separate variable here (with the conditional assignment), since the rest of the code is the same.